bigquery_magics/bigquery.py

```diff
@@ -586,6 +596,72 @@ def _handle_result(result, args):
     return result
 
 
+def _is_colab() -> bool:
+    """Check if code is running in Google Colab"""
+    try:
+        import google.colab  # noqa: F401
+
+        return True
+    except ImportError:
+        return False
+
+
+def _colab_callback(query: str, params: str):
+    return IPython.core.display.JSON(
+        graph_server.convert_graph_data(query_results=json.loads(params))
+    )
+
+
+singleton_server_thread: threading.Thread = None
+
+
+def _add_graph_widget(query_result):
+    try:
+        from spanner_graphs.graph_visualization import generate_visualization_html
+    except ImportError as err:
+        customized_error = ImportError(
+            "Use of --graph requires the spanner_graphs package to be installed."
```

Comment:
Let's be more specific about what the user needs to do.

```suggestion
            "Use of --graph requires the spanner-graph-notebook package to be installed. Install it with `pip install 'bigquery-magics[spanner-graph-notebook]'`."
```